packages/console/package.json

```diff
@@ -21,20 +25,24 @@
     "url": "https://github.com/patternfly/patternfly-react/issues"
   },
   "homepage": "https://github.com/patternfly/patternfly-react/blob/master/src/console/README.md",
-  "publishConfig": {
-    "access": "public"
+  "sassIncludes": {
+    "patternfly": "../../node_modules/patternfly/dist/sass",
+    "bootstrap": "../../node_modules/bootstrap-sass/assets/stylesheets",
+    "fontAwesome": "../../node_modules/font-awesome-sass/assets/stylesheets"
   },
   "scripts": {
-    "build": "node ../../scripts/runScript.js build:console",
-    "prepublish": "yarn build"
+    "prebuild": "yarn build:scripts && yarn build:sass && yarn build:less",
+    "build:scripts": "babel src --out-dir dist/js --ignore .test.js,__mocks__",
```

Comment:
this actually is probably not needed since the lerna build command should take care of it.